### ISSUE TITLE
Windows 10, focus

### ISSUE THREAD

**Issue by Samuel-Weiss** [broken-e/DragDropConfirm#14]
Hy, I'm trying to use DragDropConfirm on Windows 10. But I stumbled with following issue:
- move a folder to the desktop is working
- the confirm box is behind the folder windows and it is hidden. 
Does soemone know a solution for it?

**Comment by bduffek** [comment]
Sorry for this late reply.  I'm not the original creator but I messed with this a bit last year.  It's been over a year so I may remember incorrectly, but even then I think the pre-compiled version was behind the commits.

This is the modification I made:
https://github.com/bduffek/DragDropConfirm/commit/15dd49d1fbf1d90d37048748c96031af8277d44b

I don't know if your issues would still happen, but it's possible they are fixed especially if the files you downloaded were out of date.
